i18n/states.php

```diff
@@ -349,6 +349,24 @@
 	),
 	'FI' => array(),
 	'FR' => array(),
+	'GH' => array( // Ghanaian Regions
```

Comment:
Seems like that is missing two regions listed on https://github.com/unicode-org/cldr/blob/release-37/common/subdivisions/en.xml#L1845-L1864

Reply:
Sorry @claudiosanches I missed your earlier comment about the updated CLDR list. I've added the two regions